app/views/reviews/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="container top-margin" id="reviews-index">
+
+  <div class="row">
+    <div class="col-xs-8">
+      <div><strong><%= @playspace.name.titleize %></strong></div>
+      <div class="bottom-margin"><em><%= @playspace.district.titleize %>, <%= @playspace.region %></em></div>
+      <div>Address: <%= @playspace.address.titleize %></div>
+    </div>
+    <div class="col-xs-4">
+      <%= link_to 'Back', playspace_path(@playspace.id), class: "btn btn-default" %>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-xs-12 col-sm-10 col-md-8 features">
+      <h6>Features</h6>
+      <span class="features-list"><%= features_list(@playspace) %></span>
+    </div>
+  </div>
```

Comment:
i don't think you need features inside your reviews-index, features should only be in playspace-index